pytapo/__init__.py

```diff
@@ -331,6 +331,40 @@ def setPreset(self, presetID):
             raise Exception("Preset " + str(presetID) + " is not set in the app.")
         return self.performRequest({"method": "do", "preset": {"goto_preset": {"id": str(presetID)}}})
 
+    def getLensDistortionCorrection(self):
+        data = self.performRequest({"method": "get", "image": {"name": ["switch"]}})
+        return data["image"]["switch"]["ldc"] == "on"
+
+    def setLensDistortionCorrection(self, enable):
+        return self.performRequest({
+            "method": "set",
+            "image": {"switch": {"ldc": "on" if enable else "off"}}
+        })
+
+    def getImageFlipVertical(self):
+        data = self.performRequest({"method": "get", "image": {"name": ["switch"]}})
+        return data["image"]["switch"]["flip_type"] == "center"
+
+    def setImageFlipVertical(self, enable):
+        return self.performRequest({
+            "method": "set",
+            "image": {"switch": {"flip_type": "center" if enable else "off"}}
+        })
+
+    def getLightFrequencyMode(self):
+        data = self.performRequest({"method": "get", "image": {"name": ["common"]}})
+        return data["image"]["switch"]["light_freq_mode"]
```

Comment:
This function is now implemented in 1.2 as getCommonImage, I needed to figure out how to get dayNightMode when creating tests.